sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py

```diff
@@ -59,8 +59,29 @@ def order_results(response, combined):
 
 
 def prepare_result(func):
+    def _get_error_code_and_message(error):
+        if hasattr(error.error, 'innererror') and error.error.innererror:
+            return error.error.innererror.code, error.error.innererror.message
```

Comment:
lol. this is so weird.  error.error.innererror.code.

Reply:
yeah it's pretty disgusting